configs/milvus.yaml

```diff
@@ -570,6 +570,8 @@ dataCoord:
     maxParallelTaskNum: 10
     dropTolerance: 86400 # Compaction task will be cleaned after finish longer than this time(in seconds)
     gcInterval: 1800 # The time interval in seconds for compaction gc
+    scheduleInterval: 1000 # The time interval in milliseconds for scheduling compaction tasks. If the configuration setting is below 100ms, it will be ajusted upwards to 100ms
+    checkInterval: 1000 # The time interval in milliseconds for checking compaction tasks. If the configuration setting is below 100ms, it will be ajusted upwards to 100ms
```

Comment:
Before this PR: schedulerInterval and checkInteval is 3s

Not sure if we should set default to 
- 500ms as in this PR
- 3000 the same as before.